docs/quickstart/installing.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: default
-title: Install lakeFS
-description: Installing lakeFS is easy. This section covers how to spin up lakeFS using Docker.
+title: Run lakeFS Locally
+description: Running lakeFS locally is easy and quick. This section covers how to spin up lakeFS using Docker.
 parent: Quickstart
 nav_order: 10
 has_children: false
 next: ["Create your first repository", "./repository.html"]
 ---
 
-# Install lakeFS
+# Run lakeFS Locally
```

Comment:
It's better to use Install lakeFS locally instead of run, as from a developer point of view, the user essentially install it.